gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1054,7 +1057,7 @@ public void testGlobStatus() throws IOException {
     createFile(new Path("/directory1/subdirectory2/file1"), data);
     createFile(new Path("/directory1/subdirectory2/file2"), data);
 
-    FileStatus[] rootDirectories = ghfs.globStatus(new Path("/d*"));
+    FileStatus[] rootDirectories = ghfs.globStatus(new Path("/directory1*"));
```

Comment:
Why do we need to change glob pattern?

Reply:
It's because we updated the bucket prefix. As it's going to be removed from this PR this change will also go away